src/melee/ft/ftparts.c

```diff
@@ -14,7 +14,9 @@
 #include <sysdolphin/baselib/pobj.h>
 #include <sysdolphin/baselib/util.h>
 
-#define MAX_FT_PARTS 140
+#include <dolphin/os/os.h>
+
+#define MAX_FT_PARTS (140)
```

Comment:
```suggestion
#define MAX_FT_PARTS 140
```
nit, parens not necessary for literals

Reply:
Really? There are no adverse edge cases?

Reply:
Nope, parens are only necessary when you are expanding an arg in a function-like macro or when you are expanding to multiple tokens

Reply:
Ok well I'm not motivated to fix it in bulk but I'll start phasing out the parens for literals.